### ISSUE TITLE
About the pipeline to video

### ISSUE THREAD

**Issue by chikiuso** [supasorn/synthesizing_obama_network_training#6]
Hi, I train and generate the result (.txt with lots of parameters), what's next to generate a video? How could I generate a video with the result? thanks.

**Comment by Ghostpepper123** [comment]
Hi, did you figure it out? I've been trying to figure this script out too.  I'm currently trying to figure out what 'save_dir' is.  Have you made any progress?

**Comment by 0x10cxR1** [comment]
Hi @chikiuso, do you know how to deal with the txt results?
Thanks!

**Comment by dvirsegal** [comment]
joining the question

**Comment by rragundez** [comment]
@supasorn I think this is an important issue. Could you comment on it?

**Comment by supasorn** [comment]
I'll try to get the pca basis (I just moved and don't have access now) which is used to produce the sparse shape. However, the pipeline is up to 10k lines of code and I might not have the bandwidth to publish them all in the near future.

**Comment by Mu7ammad** [comment]
@supasorn Yes please, having the PCA basis to be able to reconstruct the reference mouth landmarks will add great value to this repository, even without the rest of the mouth texture pipeline. Thanks for sharing these with us, and looking forward to this last piece.

**Comment by nikhitha-m** [comment]
@supasorn  It would be a great help if you could share at least PCA basis for time being.

**Comment by chikiuso** [comment]
I got the data something like the following in the result folder, how could I turn it into a working video? thanks : 

8941 21
0.012750 -14.305814 1.723046 -3.195284 0.839943 0.292246 0.310746 -0.673671 -0.035229 -0.497246 0.038055 -0.060966 -0.281909 0.200559 -0.007483 -0.090569 0.028182 -0.071471 -0.060519 -0.064697 0.053031
0.012750 -10.957369 0.370900 -4.028065 1.322961 -0.047874 -0.381775 -0.194586 -0.195218 -0.259842 -0.151549 0.010871 -0.246901 0.094218 -0.064462 -0.098995 -0.022320 -0.059445 -0.046436 -0.051390 0.033529
0.012750 -12.626245 0.770954 -3.939007 1.471983 0.298785 -0.459137 -0.174414 -0.107329 -0.370750 -0.217055 0.006996 -0.261211 0.154793 -0.053607 -0.107184 -0.020559 -0.124198 -0.060682 -0.090156 0.025293
0.012750 -11.888833 0.430780 -4.011168 1.658940 0.089108 -0.692530 -0.073704 -0.138650 -0.369726 -0.305787 0.005497 -0.214101 0.149432 -0.047078 -0.105292 -0.043473 -0.108837 -0.038625 -0.106557 0.012551
0.012750 -12.715007 0.538861 -3.877327 1.780462 0.109226 -0.668121 -0.107567 -0.095448 -0.421308 -0.334100 -0.005160 -0.202670 0.153702 -0.046681 -0.099228 -0.042577 -0.113163 -0.025554 -0.128626 0.001390
0.012750 -5.926603 1.824336 -4.906502 -0.294607 0.156135 -0.279463 0.000935 0.388505 -0.467345 0.117154 0.056054 -0.067724 -0.023136 -0.003218 -0.000294 -0.082321 -0.062127 -0.175327 -0.092154 0.078000

**Comment by jithinsankar** [comment]
Somebody please provide the pipeline!!

**Comment by nadia-metoui** [comment]
Joining the request for the pipeline
Thanks!

**Comment by ghost** [comment]
joining request as well, anyone been able to run the whole pipeline in google colab?

**Comment by matkir** [comment]
Joining

**Comment by FraGoTe** [comment]
any update on this?

**Comment by ldtuanAPCS** [comment]
Joining

**Comment by nirvitarka** [comment]
joining the request for full pipeline, @supasorn, thank you for sharing this work ,after 5 years of this original request raised, this repository is still one of the best approach to implement the lipsync, awaiting the remaining part from you now to see the real outcome of my training
